ansible/AKS/aks_create.yml

```diff
@@ -0,0 +1,73 @@
+# Description
+# ===========
+# This playbook create an ACR, an AKS and deploy secrets into AKS
+
+- hosts: localhost
+  vars_files:
+  - aks_vars.yml
+      
+  tasks:
+  - name: Create a resource group
+    azure_rm_resourcegroup:
+     name: "{{ resource_group }}"
+     location: "{{ location }}"
+
+  - name: Create an AKS instance
+    azure_rm_aks:
+     name: "{{ resource_name }}"
+     resource_group: "{{ resource_group }}"
+     location: eastus
+     dns_prefix: "{{ resource_name }}"
+     service_principal:
+         client_id: "{{ AZURE_CLIENT_ID }}"
+         client_secret: "{{ AZURE_SECRET }}"
+     linux_profile:
+         admin_username: azureuser
+         ssh_key: "{{ SSH_PUBLIC_KEY }}"
+     agent_pool_profiles:
+         - name: default
+           count: 1
+           vm_size: Standard_DS1_v2
+    register: aks
+
+  - name: Create Azure Container Registry
```

Comment:
if i understand correctly, this is seperate scenario, one for aci, one for aks.  aks  creation seems in one playbook.  aci is seperated as acr_create.yml, aci_create.yml. why not merge acr_create.yml into aci_create.yml

Reply:
You are right. I try to make every e2e self-contained and avoid cross-references. Customers can copy the folder of a scenario and use run it directly.